docs/connectRoutes.md

```diff
@@ -53,107 +55,80 @@ type RouteObject = {
 
-* **notFoundPath** - the `notFoundPath` defaults to `'/not-found'`. The address gets redirected here in 2 situations: when you dispatch an action with no matching path, or if you manually call `dispatch(redirect({ type: NOT_FOUND }))`, where `NOT_FOUND` is an export from this package. The type in actions and state will be `NOT_FOUND`, which you can use to show a 404 page. Conversely, if the user visits a URL directly or if you dispatch `NOT_FOUND` without the redirect, the ***current URL is preserved*** but the `NOT_FOUND` type is *also* dispatched, which is the correct way websites are typically supposed to deal with URLs they don't handle. *I.e. just like on Github.com, a 404 graphic will show and the URL stays the same.* So you will *rarely see* `'/not-found'` unless you trigger it (intentionally or by accident). Lastly if you specify `notFoundPath: null`, in the aforementioned scenarios, the URL will display as the previous URL (i.e. the URL currently in the address bar) and fallback to `'/'`, such as in SSR if no history exists yet.
+* **location** - a function to specify where in your Redux state tree **Redux First Router** should expect its reducer to be attached to. This can be omitted if you attach the reducer at `state.location`. You can provide a function here which can be useful if you need to implement custom logic to get at the `location` state. This is especially useful for non-standard state shapes, such as Immutable.js. For example, if `state` is an instance of `Immutable.Map` you might have `state => state.get('location')`.
 
-* **scrollTop** - the `scrollTop` option calls `window.scrollTo(0, 0)` on route changes so the user starts each page at the top. This is a *"poor man's"* scroll
-restoration, and should be fine while developing, especially if you're using Chrome. Though hash links won't fully function. See the next option for full-on scroll restoration support.
+* **title** - a function to specify where in your Redux state tree **Redux First Router** should expect its reducer to be attached to. This can be omitted if you attach the reducer at `state.title`. **Redux First Router** will change your page title for you when the route changes, e.g. `document.title = 'foo'`. As with `location` you can provide a function here which can be useful if you need to implement custom logic to get at the title state. Example: `state => state.get('title')`.
```

Comment:
This is wrong - `title` is ont where the RFR reducer is attached, its how to get the page title from the state.

In this case I think the original is pretty concise and accuratet: "the name of the state key for your page title or a selector function for getting it from state"

Reply:
Sorry, I wanted to make a similar description to `location`, but I forgot the opportunity of a string. It was unclearly described for the latter.